src/types/interface.ts

```diff
@@ -0,0 +1,14 @@
+export type DialogOnQueue = {
```

Comment:
To not confuse it with typescript's `interface`, could we rename this file to `ui.ts`?